xml/schema/Case/CaseType.xml

```diff
@@ -52,6 +52,20 @@
     <comment>Description of the Case Type</comment>
     <add>4.5</add>
   </field>
+  <field>
+    <name>category</name>
+    <title>Category</title>
+    <type>int unsigned</type>
+    <required>true</required>
+    <add>5.14</add>
+    <comment>FK to a civicrm_option_value (case_type_category)</comment>
+    <pseudoconstant>
+      <optionGroupName>case_type_category</optionGroupName>
+    </pseudoconstant>
+    <html>
+      <type>Select</type>
+    </html>
+  </field>
```

Comment:
Needs to be multilingual aware

Reply:
How can that be achieved? Is there an example?

Reply:
found it!